Pipfile

```diff
@@ -0,0 +1,19 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
```

Comment:
Managed to get the first Url to work but any others just renders 'Please enter a URL'